content/library/advanced-features/forms.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Using forms
+slug: /library/advanced-features/forms
+---
+
+# Using forms
+
+When you don't want to rerun your script with each input made by a user, `st.form` is here to help! Forms make it easy to batch user input into a single rerun. This guide to using forms provides examples and explains how users interact with forms.
```

Comment:
Let's link to the API on first mention: [`st.form`](/library/api-reference/control-flow/st.form)